README.md

```diff
@@ -9,16 +9,17 @@ Toggle the extension on and you are ready to go.
 ## Usage
 In the console the following functions are available to you:
 
-| function         | description                               |
-| ---------------- | ----------------------------------------- |
-| getMap           | get the current map                       |
-| getMaps          | returns all the maps for your environment |
-| getPlayers       | teleports your avatar in a selected space |
-| listMaps         | prints all the mpas                       |
-| listPlayers      | teleports your avatar in a selected space |
-| position         | get the current position                  |
-| teleport         | teleports your avatar in a selected space |
-| teleportToPlayer | teleports your avatar in a selected space |
+| function           | description                               |
+| ----------------   | ----------------------------------------- |
+| getMap             | get the current map                       |
+| getMaps            | returns all the maps for your environment |
+| getPlayers         | teleports your avatar in a selected space |
+| listMaps           | prints all the mpas                       |
+| listPlayers        | teleports your avatar in a selected space |
+| position           | get the current position                  |
+| teleport           | teleports your avatar in a selected space |
+| teleportToPlayer   | teleports your avatar in a selected space |
+| teleportToSpawn    | teleports your avatar in a selected space |
```

Comment:
would just add a precision here: `teleports your avatar to a spawn on the selected space` or something like that